ibm_catalog.json

```diff
@@ -0,0 +1,146 @@
+{
+    "products": [
+      {
+        "name": "deploy-arch-ibm-mq-cloud",
+        "label": "MQ on Cloud",
+        "product_kind": "solution",
+        "tags": [
+          "ibm_created",
+          "integration",
+          "target_terraform",
+          "terraform",
+          "solution"
+        ],
+        "keywords": [
+          "MQ",
+          "MQ on Cloud",
+          "IaC",
+          "infrastructure as code",
+          "terraform",
+          "solution"
+        ],
+        "short_description": "Creates and configures IBM Cloud MQ on Cloud resources",
+        "long_description": "This architecture supports creating and configuring an IBM MQ on Cloud deployment instance and queue manager.",
+        "offering_docs_url": "https://github.com/terraform-ibm-modules/terraform-ibm-mq-cloud/blob/main/solutions/standard/README.md",
+        "offering_icon_url": "https://raw.githubusercontent.com/terraform-ibm-modules/terraform-ibm-mq-cloud/main/images/MQ.svg",
+        "provider_name": "IBM",
+        "features": [
+          {
+            "title": "Creates an MQ on Cloud deployment instance.",
+            "description": "Creates and configures an MQ on Cloud deployment Protect instance."
+          }
+        ],
+        "support_details": "This product is in the community registry, as such support is handled through the originated repo. If you experience issues please open an issue in that repository [https://github.com/terraform-ibm-modules/terraform-ibm-mq-cloud/issues](https://github.com/terraform-ibm-modules/terraform-ibm-mq-cloud/issues). Please note this product is not supported via the IBM Cloud Support Center.",
+        "flavors": [
+          {
+            "label": "Standard",
+            "name": "standard",
+            "install_type": "fullstack",
+            "working_directory": "solutions/standard",
+            "compliance": {
+              "authority": "scc-v3",
+              "profiles": [
+                {
+                  "profile_name": "CIS IBM Cloud Foundations Benchmark",
```

Comment:
Is there even any SCC rules in any profiles for anything the DA deploys? If there is not, there is no point in adding anything for compliance here (can be set to `"compliance": {}`).

Also this profile is deprecated. The latest one is called `CIS IBM Cloud Foundations Benchmark v1.1.0` (yes with the version in the name) and the version is `1.1.0`